PWGHF/TableProducer/treeCreatorD0ToKPi.cxx

```diff
@@ -401,6 +426,20 @@ struct HfTreeCreatorD0ToKPi {
       if (candidate.isSelD0bar()) {
         fillTable(candidate, 1, massD0bar, hfHelper.cosThetaStarD0bar(candidate), topolChi2PerNdf, ctD, yD, eD, 0, 0);
       }
+      if constexpr (applyMl) {
+        if (candidate.isSelD0()) {
+          rowCandidateMl(
+            candidate.mlProbD0()[0],
+            candidate.mlProbD0()[1],
+            candidate.mlProbD0()[2]);
+        }
+        if (candidate.isSelD0bar()) {
+          rowCandidateMl(
+            candidate.mlProbD0bar()[0],
+            candidate.mlProbD0bar()[1],
+            candidate.mlProbD0bar()[2]);
+        }
+      }
```

Comment:
I would suggest to move this to the `fillTable` function (adding the `applyMl` template argument), to have the filling of the tables in the same place